src/openforms/authentication/api/views.py

```diff
@@ -39,6 +40,25 @@ def get_objects(self):
         return list(register.iter_enabled_plugins())
 
 
+@extend_schema_view(
+    get=extend_schema(summary=_("List available Yivi attribute groups")),
+)
+class AttributeGroupListView(ListMixin, APIView):
```

Comment:
Additionally, this views belongs in the yivi app, not the generic app that's plugin-agnostic.